iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -380,8 +419,21 @@ static LogicalResult setRootConfig(
     // number of loops when adding the pass to pass manager.
     // TODO(hanchung): Embed options into attributes, so we can control options
     // more heuristically.
-    return setX86SandboxRootConfig(entryPointFn, contractionOp,
-                                   workloadPerWorkgroup, vectorSize);
+    int lhsElemBitwidth = getElementTypeOrSelf(contractionOp.lhs().getType())
```

Comment:
Do you need to check the bit width? How about just doing equality check on the element type itself.

Reply:
ah, good point.